website/static/js/_allLogTexts.js

```diff
@@ -0,0 +1,76 @@
+var logActions = {
+    'project_created' : '${user} created ${node}', 
```

Comment:
Minor: Trim trailing whitespace. Some editors and linters consider this a linting violation.